avm/res/cache/redis/linked-servers/main.bicep

```diff
@@ -0,0 +1,39 @@
+metadata name = 'Redis Cache Linked Servers'
+metadata description = 'This module connects a primary and secondary Redis Cache together for geo-replication.'
+metadata owner = 'Azure/module-maintainers'
+
+@description('Required. Primary Redis cache name.')
+param primaryRedisCacheName string
+
+@description('Required. The name of the secondary Redis cache. If not provided, the primary Redis cache name is used.')
+param secondaryRedisCacheName string
+
+@description('Required. The resource ID of the linked server. If not provided, the resource ID of the primary Redis cache is used.')
+param linkedRedisCacheId string
+
+@description('Required. The location of the linked server. If not provided, the location of the primary Redis cache is used.')
+param linkedRedisCacheLocation string
```

Comment:
'If not provided' isn't possible as the parameter is currently required. However, you can make it optional and aligned with the description if you apply this [suggestion](https://github.com/Azure/bicep-registry-modules/pull/2562/files#r1658289706) make this parameter nullable, and where `linkedRedisCacheLocation` is used you write: `linkedRedisCacheLocation: linkedRedisCacheLocation ?? redisCache.location`

Reply:
I think I've got this updated.